recipes/libcufft/meta.yaml

```
@@ -0,0 +1,137 @@
+{% set name = "libcufft-split" %}
+{% set version = "11.0.0.21" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/libcufft/{{ platform }}/libcufft-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 4650f79fcf377af410afedb56445f1332e8f213ec3c94d660f9a91328c3ecb4a  # [linux64]
+  sha256: e711671995e9d92d16015a4cd3751ec30710a49fa22f2b88a4a42905c248d861  # [ppc64le]
+  sha256: 91aa1ba5e68ca65778222c9a427a9214c4550d2f45e978e51779a207bafc17a3  # [aarch64]
+  sha256: a33f577d48ef432d5cfa8d0ebfa789cc5261af5dd84fd9c608235edc3f769e43  # [win]
+
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+
+script: install.sh   # [linux]
+script: install.bat    # [win]
+
+
+outputs:
+  - name: libcufft
+    files:
+      - lib/libcufft*.so.*    # [linux]
+      - targets/*/lib/*.so.*  # [linux]
+      - Library\bin           # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+      host:                                    # [linux]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+        - __glibc >=2.17                       # [linux]
+      run:                                     # [linux]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+        - __glibc >=2.17                       # [linux]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufft.so.{{ version }}                              # [linux]
+        - if not exist %LIBRARY_BIN%\cufft64_{{ version.split(".")[0] }}.dll exit 1  # [win]
+
+  - name: libcufft-dev
+    files:                   # [linux]
+      - lib/*.so             # [linux]
+      - lib/stubs            # [linux]
+      - include              # [linux]
+      - lib/pkgconfig        # [linux]
+      # Windows is only one target; everything is in `...-dev_target`
+    run_exports:
+      weak:
+        - {{ pin_subpackage("libcufft", max_pin='x') }}
+    requirements:
+      run:
+        - libcufft-dev_{{ target_platform }} =={{ version }}
+        # TODO: Do we need cudart?  But at this point it would fail CI.
+      run_contrained:
+        # Ensure static package is compatible (if installed explicitly)
+        - libcufft-static_{{ target_platform }} >={{ version }}  # [linux]
```

Note there's a typo fix in `run_constrained` that might escape the eye.

```suggestion
    requirements:
      host:
        - cuda-version {{ cuda_version }}
      run:
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
        - libcufft-dev_{{ target_platform }} =={{ version }}
        # TODO: Do we need cudart?  But at this point it would fail CI.
      run_constrained:
        # Ensure static package is compatible (if installed explicitly)
        - libcufft-static_{{ target_platform }} >={{ version }}  # [linux]
```